tests/conftest.py

```diff
@@ -22,22 +22,24 @@ def __init__(self, *args, **kwargs):
     @Request.application
     def __call__(self, request):
         try:
-            status, content = self.responses[len(self.queries)]
+            response = self.responses[len(self.queries)]
         except IndexError:
             response = Response("No prepared response")
             response.status_code = httplib.NOT_IMPLEMENTED
             return response
 
         self.queries.append(request)
+        return response
 
-        response = Response(json.dumps(content))
+    def add_response(self, data=None, status=httplib.NO_CONTENT, headers=None):
```

Comment:
You mean for `data`, so it should always be handed in explicitly? I thought about it and I actually like it.